### ISSUE TITLE
wk4 june_epic BUG: Stuck at SIM UI Login after clicking "Select World" for either Team or Individual world

### ISSUE THREAD

**Issue by lijenn** [lzim/teampsd#2073]
Identified during live team meeting at 10:30a Pacific on Wednesday 6/23 with @dlkibbe @clairecollie :

@lijenn will ping James the specific user, started to occur around 11:15am Pacific.
The user gets stuck here after clicking "Select World". 
![image](https://user-images.githubusercontent.com/59668647/123149046-b3bf6a00-d415-11eb-9821-a67461c8aeee.png)
The Ghostbusters logo/red circle with the cross logo comes up but gets stuck and doesn't sign the user into their Team or Individual world.

**Comment by jamesmrollins** [comment]
Hi @lijenn 

- Is this the first time the user tried to login? 
- Which browser is the user using?

**Comment by jamesmrollins** [comment]
@lijenn 

I logged in as the user and was not able to recreate the issue.  However, I do notice that the user login for the _ind world is as a "member" instead of a "team lead."  I will follow up with DEV about that.  As a precautionary measure, I did unassign and then reassign the user to the team and changed the roles for everyone in the _ind team.  Hopefully this resolves the issue.  I think we should ask the user to try again and see what happens.  If it happens again, attempt the login using the incognito function if possible. 

I checked the SmartLook videos and they appear to show the user logging in without any issues, so I am a little confused.
